src/pages/guardian-directory.tsx

```diff
@@ -31,30 +64,43 @@ const GuardianDirectory = () => {
   return loading ? (
     <Loader />
   ) : (
-    <div className={styles.guardianDirectory}>
-      <h1>Guardian Directory</h1>
-      <GuardianDirectoryTable
-        headers={[
-          'First Name',
-          'Last Name',
-          'Rank',
-          'Title',
-          'Base',
-          'Field Commands',
-          'Email',
-        ]}
-        keys={[
-          'FirstName',
-          'LastName',
-          'Rank',
-          'DutyTitle',
-          'BaseLoc',
-          'MajCom',
-          'Email',
-        ]}
-        rows={directory}
-      />
-    </div>
+    <>
+      <div className={styles.search}>
+        <Search
+          onSubmit={searchDirectory}
+          placeholder="Search Guardian Directory"
+          aria-label="Search Guardian Directory"
+          className={styles.searchField}
+        />
+        <Button type="button" unstyled onClick={resetSearch}>
```

Comment:
There wasn't a reset in the design, but it felt necessary to have a way to reload the directory without reloading the page. Consider this a placeholder. cc @shkeating @AnnaGingle for further feedback and changes.